src/main/daml/Daml/Finance/Interface/Types/Date/FixingDates.daml

```diff
@@ -1,21 +1,28 @@
 -- Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 -- SPDX-License-Identifier: Apache-2.0
 
-module Daml.Finance.Interface.Instrument.Bond.Types where
+module Daml.Finance.Interface.Types.Date.FixingDates where
```

Comment:
The types define in this module aren't specific for `FixingDate`, but rather contain information on how to define a `DateOffset`.

In our scenario, this `DateOffset` is used to calculate fixing dates, but could in principle be used for payment dates as well.

What about renaming the module to `DateOffset`?